content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md

```diff
@@ -465,6 +465,12 @@ jobs:
 
 If you want to allow maintainers to mark certain pull requests for auto-merge, you can use {% data variables.product.prodname_dotcom %}'s auto-merge functionality. This enables the pull request to be merged when all required tests and approvals are successfully met. For more information on auto-merge, see "[Automatically merging a pull request](/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)."
 
+{% warning %}
+
+You have to protect the pull request's target branch to wait for all checks to pass by enabling [Require status checks to pass before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule). Without waiting for the checks, the Dependabot auto-merge will cause an instant merge into the pull request's target branch.
+
+{% endwarning %}
+
```

Comment:
There are several types of branch protection rules that you might want to enable, in addition to "Require status checks to pass before merging". So I've updated the paragraph above to clarify the expected behavior and I'm going to modify this warning according to our style guide.

```suggestion
{% note %}

**Note:** If you use status checks to test pull requests, you should enable **Require status checks to pass before merging** for the target branch for {% data variables.product.prodname_dependabot %} pull requests. This branch protection rule ensures that pull requests are not merged unless all the required status checks pass. For more information, see "[Managing a branch protection rule](/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)."

{% endnote %}

```